src/tests/integration/test-data/show/SetUpDeployTest.test_extcodesize().expected

```diff
@@ -5,7 +5,7 @@
 │   callDepth: 0
 │   statusCode: STATUSCODE:StatusCode
 │
-│  (713 steps)
+│  (712 steps)
```

Comment:
This is a pretty strange change..... Why are we suddenly taking fewer steps but getting the same result? Is this a new bug in the backend? @geo2a @jberthold

Reply:
It is also possible that the semantics itself changed slightly within the time it was unupstreamed and is genuinely taking one fewer step in that particular case.

Reply:
I'd be surprised that we did fewer steps on the same semantics. If we are concerned/see it it multiple places, I can investigate. Please open an issue if so.

Reply:
@dwightguth did an [adjustment to the semantics](https://github.com/runtimeverification/evm-semantics/pull/2310) that removed a one-step `#checkNonceExceeded` check from `#checkCall`. It's with high probability that that's what we are seeing since it's a one-step reduction.